src/licensedcode/data/licenses/st-microelectronics.LICENSE

```diff
@@ -0,0 +1,34 @@
+Copyright (c) 2017  STMicroelectronics – All rights reserved
```

Comment:
Can you remove this copyright line from the reference license text? We want to avoid mixing copyrights in licenses in most cases.